xml/net_proxy.xml

```diff
@@ -1172,29 +1164,29 @@ http_access deny all
   </para>
 
   <para>
-   The usual way of working with proxy servers is as follows: the Web browser
+   The standard way of working with proxy servers is as follows: the Web browser
    sends requests to a certain port of the proxy server and the proxy always
    provides these required objects, regardless of whether they are in its
-   cache. However, in some cases the transparent proxy mode of Squid makes
+   cache. However, in some cases using the transparent proxy mode of Squid makes
    sense:
   </para>
 
   <itemizedlist>
    <listitem>
     <para>
-     If, for security reasons, it is recommended that all clients use a proxy
-     server to surf the Internet.
+     When it is desirable that all clients use a proxy server to access the
```

Comment:
* Because it removes "security reasons", this rewording makes the first and the second point sound almost the same. Imo not good.
* Are you sure about "when" for these conditions? I am not 100% on this, but I'd probably keep "If..." (x3)